lib/curly/rspec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rspec/rails'
+
+module Curly
+  module RSpec
+    module PresenterExampleGroup
+      extend ActiveSupport::Concern
+      include ::RSpec::Rails::ViewExampleGroup
+
+      included do
```

Comment:
Yeah – we got a dependency on Rails anyway, so we might as well use the helpful bits.